components/query/partials/infopanel.html

```diff
@@ -1,22 +1,22 @@
 <hs-panel-header name="info" title="Info"></hs-panel-header>
-<div class="card panel-default hs-main-panel" ng-show="!data.coordinates[0].name.length > 0">
-    <div class="card-body m-2" translate="">Click on map for query results!</div>
+<div class="card panel- hs-main-panel" *ngIf="HsQueryBaseService.data.coordinates.length == 0">
```

Comment:
great! But `HsQueryBaseService.data.coordinates.length == 0` won't catch `HsQueryBaseService.data.coordinates.length == undefined`. It might be OK, if the undefined cannot happen by other means.

Reply:
Array HsQueryBaseService.data.coordinates: [] will always be defined, because it is being instantiated  on service initialization.   But if check also for undefined is needed then only this works => HsQueryBaseService.data.coordinates === undefined || HsQueryBaseService.data.coordinates.length == 0